### PR TITLE
Add Temporal Fragmentation to streaming appendix

### LINKED ISSUE #208: Appendix L "Streaming TTML Content" does not reflect best industry practice.
Support for streaming pre-recorded content generally requires fragmentation of a larger document, possibly one for an entire piece of content. Industry practice fragmentation techniques, such as those described in EBU Tech 3381, are inconsistent with this. Suggest updating this Appendix to reflect best industry practice, or alternatively delete it (it is not cited in the Rec body).

### PR DESCRIPTION
**Pull request by nigelmegitt**
Close #208.

* Remove or reword contentious statements about TTML design for
streamability
* Add an almost identical appendix to that in TTML2 describing temporal
fragmentation. Differs in term references, since some TTML2 terms are
not present in TTML1.
* Adds ISOBMFF and EBU-TT Live non-normative references required by new
appendix text.

### CODE REVIEW

**Comment by nigelmegitt** [comment]
Built version can be reviewed at https://rawgit.com/w3c/ttml1/issue-208-streaming-ttml-build/index.html#streaming

**Review by palemieux — APPROVED** [review]
LGTM

File: spec/ttml1.xml [hunk 1]
```diff
@@ -13186,35 +13191,29 @@ TTML <loc href="#doctypes">Abstract Document Instance</loc>.</p>
...
 possible a temporal ordering of content that follows temporal
 presentation order;</p>
 </item>
-<item>
-<p>constrains content models to prevent arbitrary nested content
-structures;</p>
-</item>
 </ulist>
+<div2 id="root-fragmentation">
+<head>Root and branch fragmentation</head>
```
Comment (skynavga) [inline spec/ttml1.xml]: Change to "Root and Branch Fragmentation".

File: spec/ttml1.xml [hunk 2]
```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
 <p>This specification does not define a transport buffer model or a decoder
 capabilities model.</p>
 </note>
+</div2>
+<div2 id="temporal-fragmentation">
+<head>Temporal fragmentation</head>
```
Comment (skynavga) [inline spec/ttml1.xml]: Change to "Temporal Fragmentation".

File: spec/ttml1.xml [hunk 3]
```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
...
+<p>Another possible means by which TTML Content may be streamed is to
```
Comment (skynavga) [inline spec/ttml1.xml]: Remove "possible".

File: spec/ttml1.xml [hunk 4]
```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
...
+partition a <emph>Document Instance</emph>'s information set into
```
Comment (skynavga) [inline spec/ttml1.xml]: Change ``<emph>Document Instance</emph>`` to a ``<loc ...>document instance</loc>`` link.

File: spec/ttml1.xml [hunk 5]
```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
...
+temporally bound fragments, each of which is itself a document instance that contains all the
+front matter (head) and content required to present it, where the temporal interval for each
+fragment is constrained, either by the <emph>Document Interchange Context</emph> or by the 
+<emph>Document Processing Context</emph> or by both, such that temporal overlaps with other
```
Comment (skynavga) [inline spec/ttml1.xml]: Change ``<emph>Document Processing Context</emph>`` to a link to terminology section (with lower case).

File: spec/ttml1.xml [hunk 6]
```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
...
+front matter (head) and content required to present it, where the temporal interval for each
+fragment is constrained, either by the <emph>Document Interchange Context</emph> or by the 
+<emph>Document Processing Context</emph> or by both, such that temporal overlaps with other 
+fragments are resolved.</p>
+<p>For example, some document processing contexts resolve temporal overlap by specifying that a
+maximum of one document instance can be active at any moment in the timeline, and additionally
+specify precedence rules to establish which document instance that is, in case there are multiple
+candidates.</p>
+<p>In order to ensure that all the content is correctly displayed, any content that falls into an
+Intermediate Synchronic Document (ISD) whose interval overlaps with more
```
Comment (skynavga) [inline spec/ttml1.xml]: Change ``Intermediate Synchronic Document (ISD)</emph>`` to a link to terminology section (with lowercase).

**Review by skynavga — CHANGES_REQUESTED** [review]

**Comment by palemieux** [comment]
@nigelmegitt Let me know if you would like me to implement the suggested editorial tweaks above.

File: spec/ttml1.xml (hunk 3 above)
Comment (nigelmegitt) [reply spec/ttml1.xml]: This is the same wording as in the opening text of the root and branch fragmentation section.

**Comment by nigelmegitt** [comment]
@skynavga comments addressed. You might want to open an issue for those editorial points in w3c/ttml2 too, since the same text is present there, with the same capitalisation. 

@palemieux thanks for the offer, no need this time.

**Comment by nigelmegitt** [comment]
Sorry for the noise, I don't know why 50a591f exists - some mysterious GitHubbery that appears harmless.

**Review by skynavga — APPROVED** [review]
Thanks. Yes, the same change needs to go into TTML2, which I missed at the time.

**Comment by nigelmegitt** [comment]
@skynavga Do you also want to remove the "possible" in the root and branch fragmentation opening paragraph, within this pull request?

**Comment by skynavga** [comment]
I approve the current text as found in 50a591f6de6fdf5fb78e355174c9a764c74ed1d5.

**Pull request merged**